src/pybind/mgr/cephadm/module.py

```diff
@@ -2308,6 +2314,96 @@ def _generate_prometheus_config(self):
         })
         return j
 
+    def _generate_grafana_config(self):
```

Comment:
What about moving this to a new python file? module.py is already way too big.

Reply:
+1 but I'd like to get this PR in first and then outsource this method from both modules into `common.py` in separate PR, if you don't mind.

Reply:
ok with me. (just moving code is a bit harder that having code in the correct file in the first place)

Reply:
I understand, but I'm afraid changes in the dashboard would result in this PR from being merged rather later than sooner. And since cephadm is under heavy development, I'd prefer to have it merged sooner.